Python/Product/Analysis/Interpreter/Ast/AstPythonModule.cs

```diff
@@ -76,6 +76,15 @@ string moduleFullName
             );
         }
 
+        public static IPythonModule FromTypeStub(
+            IPythonInterpreter interpreter,
+            string stubFile,
+            PythonLanguageVersion langVersion,
+            string moduleFullName
+        ) {
```

Comment:
The signature intentionally matches the other creation functions for consistency and so we can support requiring those arguments in the future without changing the public API.